test/Directory.Build.props

```diff
@@ -35,6 +35,7 @@
 
   <PropertyGroup Condition="'$(Coverage)' == 'true'">
       <CollectCoverage>true</CollectCoverage>
+      <SingleHit>true</SingleHit>
```

Comment:
This I guess is the major change, where you just register whether a line is hit or not, vs. count of hits, etc.?

Reply:
yep, he made this change in the coverlet tool! Thanks, Sam!

---
In reply to: [262544802](https://github.com/dotnet/machinelearning/pull/2843#discussion_r262544802) [](ancestors = 262544802)